pkg/cmd/project/list/list.go

```diff
@@ -0,0 +1,53 @@
+package list
```

Comment:
TODO some tests?

Reply:
still getting to that

Reply:
actually not required for simple list commands